# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html.ini
# Commit: e55eb174b342
# Full Hash: e55eb174b342c4ded8f84304980a866ba810eabd
# Author: Peter Van der Beken <peterv@propagandism.org>
# Date: 2021-12-16 16:31:55
# Regressor Bug: 1742865
# File Overlap Count: 1
# Description:
#   Bug 1742865 - Handle meta refresh correctly with session history in the parent. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D132837
# ==============================================================================

diff -r cfb3ff7ce730 -r e55eb174b342 testing/web-platform/meta/html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html.ini
--- a/testing/web-platform/meta/html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html.ini	Thu Dec 16 11:12:53 2021 +0200
+++ b/testing/web-platform/meta/html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/parsing.html.ini	Thu Dec 16 08:19:45 2021 +0000
@@ -43,13 +43,6 @@
       if debug and (os == "mac"): FAIL
       if not debug and (os == "win") and (version == "6.1.7601"): FAIL
 
-  ["-0; url=foo"]
-    disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
-    expected:
-      if not debug and (os == "mac") and (processor == "x86") and (bits == 32): FAIL
-      if debug and (os == "mac"): FAIL
-      if not debug and (os == "win") and (version == "6.1.7601"): FAIL
-
   ["+1; foo"]
     disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
     expected:
@@ -64,13 +57,6 @@
       if debug and (os == "mac"): FAIL
       if not debug and (os == "win") and (version == "6.1.7601"): FAIL
 
-  ["-0; foo"]
-    disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
-    expected:
-      if not debug and (os == "mac") and (processor == "x86") and (bits == 32): FAIL
-      if debug and (os == "mac"): FAIL
-      if not debug and (os == "win") and (version == "6.1.7601"): FAIL
-
   ["+1"]
     disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
     expected:
@@ -79,14 +65,6 @@
       if not debug and (os == "win") and (version == "6.1.7601"): FAIL
       if (processor == "x86_64") and (bits == 64): FAIL
 
-  ["-1"]
-    disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
-    expected:
-      if not debug and (os == "mac") and (processor == "x86") and (bits == 32): FAIL
-      if debug and (os == "mac"): FAIL
-      if not debug and (os == "win") and (version == "6.1.7601"): FAIL
-      if (processor == "x86_64") and (bits == 64): FAIL
-
   ["+0"]
     disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
     expected:
@@ -95,13 +73,6 @@
       if not debug and (os == "win") and (version == "6.1.7601"): FAIL
       if (processor == "x86_64") and (bits == 64): FAIL
 
-  ["-0"]
-    disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
-    expected:
-      if not debug and (os == "mac") and (processor == "x86") and (bits == 32): FAIL
-      if debug and (os == "mac"): FAIL
-      if not debug and (os == "win") and (version == "6.1.7601"): FAIL
-
   [".9; url=foo"]
     disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1230909
     expected:
@@ -115,9 +86,6 @@
   [",foo"]
     expected: FAIL
 
-  ["-0.1; url=foo"]
-    expected: FAIL
-
   [<meta>: "1\\f"]
     expected: TIMEOUT
 
@@ -172,12 +140,6 @@
   [Refresh header: "+0; url=foo"]
     expected: FAIL
 
-  [<meta>: "-0; url=foo"]
-    expected: FAIL
-
-  [Refresh header: "-0; url=foo"]
-    expected: FAIL
-
   [<meta>: "+1; foo"]
     expected: FAIL
 
@@ -190,42 +152,18 @@
   [Refresh header: "+0; foo"]
     expected: FAIL
 
-  [<meta>: "-0; foo"]
-    expected: FAIL
-
-  [Refresh header: "-0; foo"]
-    expected: FAIL
-
   [<meta>: "+1"]
     expected: FAIL
 
   [Refresh header: "+1"]
     expected: FAIL
 
-  [<meta>: "-1"]
-    expected: FAIL
-
-  [Refresh header: "-1"]
-    expected: FAIL
-
   [<meta>: "+0"]
     expected: FAIL
 
   [Refresh header: "+0"]
     expected: FAIL
 
-  [<meta>: "-0"]
-    expected: FAIL
-
-  [Refresh header: "-0"]
-    expected: FAIL
-
-  [<meta>: "-0.1; url=foo"]
-    expected: FAIL
-
-  [Refresh header: "-0.1; url=foo"]
-    expected: FAIL
-
 
 [parsing.html?11-20]
   expected: TIMEOUT
@@ -268,15 +206,7 @@
     expected: TIMEOUT
 
 
-[parsing.html?111-120]
-  [Refresh header: "-0"]
-    expected: FAIL
-
-
 [parsing.html?91-100]
-  [Refresh header: "-0; url=foo"]
-    expected: FAIL
-
   [<meta>: "+1; foo"]
     expected: FAIL
 
@@ -289,9 +219,6 @@
   [Refresh header: "+0; foo"]
     expected: FAIL
 
-  [<meta>: "-0; foo"]
-    expected: FAIL
-
 
 [parsing.html?61-70]
   [<meta>: "1; url=\\"foo'bar"]
@@ -317,43 +244,20 @@
   [Refresh header: "+0; url=foo"]
     expected: FAIL
 
-  [<meta>: "-0; url=foo"]
-    expected: FAIL
-
-
-[parsing.html?131-last]
-  [<meta>: "-0.1; url=foo"]
-    expected: FAIL
-
-  [Refresh header: "-0.1; url=foo"]
-    expected: FAIL
-
 
 [parsing.html?101-110]
-  [Refresh header: "-0; foo"]
-    expected: FAIL
-
   [<meta>: "+1"]
     expected: FAIL
 
   [Refresh header: "+1"]
     expected: FAIL
 
-  [<meta>: "-1"]
-    expected: FAIL
-
-  [Refresh header: "-1"]
-    expected: FAIL
-
   [<meta>: "+0"]
     expected: FAIL
 
   [Refresh header: "+0"]
     expected: FAIL
 
-  [<meta>: "-0"]
-    expected: FAIL
-
 
 [parsing.html?1-10]
 
